ethdev/ethdev.go

```diff
@@ -14,6 +14,17 @@ package ethdev
 #include <rte_errno.h>
 #include <rte_memory.h>
 #include <rte_ethdev.h>
+#include <rte_version.h>
+
+#if RTE_VERSION < RTE_VERSION_NUM(21, 11, 0, 0)
```

Comment:
you sure about 21.11? it'd be nice to put exact commit id as a comment here for descendants to behold...